### PR TITLE
add FireBase setup

### PR DESCRIPTION
**Pull request by pmcintyr**
add FireBase setup & configuration to the Unlost android app

### CODE REVIEW

File: app/build.gradle [hunk 1]
```diff
@@ -111,4 +117,6 @@ task jacocoTestReport(type: JacocoReport, dependsOn: ['testDebugUnitTest', 'crea
 
 connectedCheck {
     finalizedBy jacocoTestReport
-}
\ No newline at end of file
+}
+
+apply plugin: 'com.google.gms.google-services'
```
Comment (omarelmalki) [inline app/build.gradle:122]: It would be great if this line is at the top of the file right after the other plugins.

**Review by omarelmalki — APPROVED** [review]
Approved. Thanks a lot for adding it early.
Feel free to check out my comment so we can keep the build.gradle file organized.

File: app/build.gradle (hunk 1 above)
Comment (pmcintyr) [reply app/build.gradle]: done✅

**Review by ha2san — COMMENTED** [review]
Hi I see that you added some dependencies, I don't see any bug in the code but what is Firebase and why do we need it ?

**Comment by pmcintyr** [comment]
> Hi I see that you added some dependencies, I don't see any bug in the code but what is Firebase and why do we need it ?

Hi there! We are using Cloud Firestore which is a part of Firebase that we use to store and sync our app's data in a NoSQL database!

**Review by ha2san — APPROVED** [review]
> Hi there! We are using Cloud Firestore which is a part of Firebase that we use to store and sync our app's data in a NoSQL database!

Very interesting!! I don't see any problem with that, can't wait to see the progress of this stage

**Pull request merged**